.gitignore

```diff
@@ -1,15 +1,12 @@
 *.pyc
 _*
 !__init__.py
```

Comment:
Can you revert this as this seems be not related to this PR.

Reply:
This is preventing adding changes to checked-in GitHub Actions in the `.github` folder.